openapi/listings/listings.yaml

```diff
@@ -32,7 +33,7 @@ paths:
         status: proposed
       description: |-
         [Status](https://developers.vendasta.com/platform/ZG9jOjEwMTU2NTYy-versioning): `Proposed`
-        Gets a list of listings for a business location. This corresponds to the Listing Sync page in Listing Builder.
+        Gets a list of listings for a listing profile. This corresponds to the Listing Sync page in Listing Builder.
```

Comment:
This corresponds to the Listing Sync page in Listing Builder.

should be

This corresponds to the Business Profile page in Listing Builder.